.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
     {
       "mode": "auto"
     }
-  ]
+  ],
+  "deno.enable": true
```

Comment:
I think this .vscode/settings.json should be nexted in the proper apps folder right?